includes/wikia/tasks/Tasks/MultiTask.class.php

```diff
@@ -214,6 +220,12 @@ private function getWikisWherePageExists( \Title $title ) {
 			$wikis[] = intval( $row->wiki_id );
 		}
 
+		if ( empty( $wikis ) ) {
+			$this->error( 'No wikis were found that have the provided page', [
```

Comment:
In case of an DB error `DBQueryError` exception will be thrown. I'll change the message level to warning.